Committees/National_Political_Education_Committee.adoc

```diff
@@ -0,0 +1,29 @@
+= National Political Education Committee
+
+== Purpose
+
+Political education of PLTCom members is the sole mission of this committee. It will provide all members a course in capitalism first and scientific socialism second followed by a course on party work. It will also maintain a comprehensive digital library of marxist political material as well as a physical library as possible. 
+
+== Structure
+
+Only _full_ members can hold these positions:

```

Comment:
> Only _full_ members can hold these positions:

Are we differentiating membership in phase 1?

Reply:
Since education was a requirement and is now removed until the courses are developed, I don't think we can

Reply:
If we're stipulating that the education requirements come into effect once the courses are developed, we can add the same footnote in here.

Reply:
Ok sounds good